test/unit/logger-spec.js

```diff
@@ -139,171 +139,6 @@ describe("RiseVision.Common.LoggerUtils", function() {
     });
   });
 
-  describe("setIds", function() {
```

Comment:
@santiagonoguez Should we not set these tests to be skipped for now instead of completely removing them given we are not completely removing the client side logging code base?

Reply:
@stulees understood, I'll skip them instead !

Reply:
@stulees updated !